remsfal-service/src/test/java/de/remsfal/service/control/ZeebeControllerTest.java

```diff
@@ -0,0 +1,60 @@
+package de.remsfal.service.control;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import jakarta.ws.rs.core.Response;
+
+import static org.mockito.Mockito.*;
+import static org.junit.jupiter.api.Assertions.*;
+
+import io.camunda.zeebe.client.ZeebeClient;
+import org.jboss.logging.Logger;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Unit tests for the ZeebeController class without using io.camunda.zeebe.client.api classes.
+ */
+class ZeebeControllerTest {
+
+    @Mock
```

Comment:
The main logic is mocked, may be because devservices are not enabled while testing.